docs/error-summary.md

```diff
@@ -132,53 +128,83 @@ use Yiisoft\Form\Widget\Text;
-    <input type="text" id="testform-name" name="TestForm[name]">
+<form action="widgets" method="POST" _csrf="vWob09HzPhcTDuhOHVU71VJQAAymEm2Hysn_8QN1Y8qOXWK9tKd9J2RYsDd8DVqNJGcxduVIAvOMrq3FSjQpoQ==">
+    <input type="hidden" name="_csrf" value="vWob09HzPhcTDuhOHVU71VJQAAymEm2Hysn_8QN1Y8qOXWK9tKd9J2RYsDd8DVqNJGcxduVIAvOMrq3FSjQpoQ==">
+    <input type="text" id="testform-name" name="TestForm[name]" minlength="4">
     <input type="text" id="testform-email" name="TestForm[email]">
-    <div class="has-text-danger" style="display:none">Custom Header:<ul></ul>Custom Footer:</div>
-    <hr class="mt-3">
-    <div>
-        <input type="submit" id="submit-20289061442001" class="button is-block is-info is-fullwidth" name="submit-20289061442001" value="Save">
+    <div class="has-text-danger">
+        <p>Custom Header:</p>
+        <ul>
+            <li>This value is not a valid email address.</li>
+            <li>Is too short.</li>
+        </ul>
+        <p>Custom Footer:</p>
     </div>
+    <hr class="mt-3">
+    <input type="submit" id="w1-submit" class="button is-block is-info is-fullwidth" name="w1-submit" value="Save">
 </form>
 ```
 
+### Only attributes
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Form\FormModelInterface;
+use Yiisoft\Form\Widget\ErrorSummary;
+use Yiisoft\Form\Widget\Field;
+use Yiisoft\Form\Widget\Form;
+use Yiisoft\Form\Widget\Text;
+
+/**
+ * @var FormModelInterface $data
+ * @var object $csrf
+ */
+?>
+
+<?= Form::widget()->action('widgets')->csrf($csrf)->begin(); ?>
+    <?= Text::widget()->for($formModel, 'name') ?>
+    <?= Text::widget()->for($formModel, 'email') ?>
+    <?= ErrorSummary::widget()->attributes(['class' => 'has-text-danger'])->onlyAttributes('name')->model($formModel) ?>
+    <hr class="mt-3">
+    <?= Field::widget()->submitButton(['class' => 'button is-block is-info is-fullwidth', 'value' => 'Save']); ?>
+<?= Form::end(); ?>
+```
+
+
 That would generate the following code after validation:
 
 ```html
-<form action="widgets" method="POST" _csrf="PwqhEiBBULqjUwjVQFBIGDt2iral9LmaJNOkfEJe-BcMPdh8RRUTitQFUKwhCClATUG7zOau1u5itPZICx-yfA==">
-    <input type="hidden" name="_csrf" value="PwqhEiBBULqjUwjVQFBIGDt2iral9LmaJNOkfEJe-BcMPdh8RRUTitQFUKwhCClATUG7zOau1u5itPZICx-yfA==">
-    <input type="text" id="testform-name" name="TestForm[name]">
+<form action="widgets" method="POST" _csrf="vWob09HzPhcTDuhOHVU71VJQAAymEm2Hysn_8QN1Y8qOXWK9tKd9J2RYsDd8DVqNJGcxduVIAvOMrq3FSjQpoQ==">
+    <input type="hidden" name="_csrf" value="vWob09HzPhcTDuhOHVU71VJQAAymEm2Hysn_8QN1Y8qOXWK9tKd9J2RYsDd8DVqNJGcxduVIAvOMrq3FSjQpoQ==">
+    <input type="text" id="testform-name" name="TestForm[name]" minlength="4">
     <input type="text" id="testform-email" name="TestForm[email]">
     <div class="has-text-danger">
-        Custom Header:
+        <p>Please fix the following errors:</p>
```

Comment:
Seems, here should output errors for "name", but not displayed error for "email".

Reply:
thks, done.